ci-operator/jobs/infra-image-mirroring.yaml

```diff
@@ -257,6 +257,14 @@ periodics:
     ci.openshift.io/area: openshift-kni
     ci.openshift.io/role: image-mirroring
   name: periodic-image-mirroring-openshift-kni
+  reporter_config:
+    slack:
+      channel: '#telco-platform-alerts'
```

Comment:
it seems to me this will alert on the telco channel for all the mirroring failures, including those which are not managed by telco teams. Is this right?

Reply:
any failure inmirroring the images in https://github.com/openshift/release/tree/c7b1c3f66feed64b2352e3b50f28c0379c492f95/core-services/image-mirroring/openshift-kni will be reported to the channel, yes.
are you suggesting that we rename the channel?

Reply:
no, it's ok if limited only to `openshift-kni` images.